.github/workflows/workflow.yml

```diff
@@ -0,0 +1,77 @@
+name: Test and Deploy
```

Comment:
Just for consistency with the other projects, can you rename the file to `ci.yml` and the workflow name to `ci`?